ironfish/src/rpc/routes/chain/getBlock.ts

```diff
@@ -148,14 +132,31 @@ routes.register<typeof GetBlockRequestSchema, GetBlockResponse>(
     const transactions: GetBlockResponse['block']['transactions'] = []
 
     for (const tx of block.transactions) {
-      const fee = tx.fee()
-
       transactions.push({
-        signature: tx.transactionSignature().toString('hex'),
```

Comment:
Can we add this signature field to `RpcTransaction` for better backwards compatibility of this endpoint

Reply:
sure